app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -396,6 +398,25 @@ class CommonTriageQuestionsController @Inject() (
       }
     }
 
+  private def amendReturnDisposalDateBackLink(state: Either[StartingNewDraftReturn, FillingOutReturn]): Call =
+    state.fold(
+      _ => controllers.routes.StartController.start(),
+      _.draftReturn.fold(
+        _ => routes.MultipleDisposalsTriageController.whenWereContractsExchanged(),
```

Comment:
this is the endpoint for completion date for multiple disposals not disposal date. The case for both multiple and multiple mixed use should come from the disposal date question in the example property details section